utils/hashers.hh

```diff
@@ -11,6 +11,12 @@
 #include "bytes.hh"
 #include "utils/hashing.hh"
 
+template<typename H>
+concept BytesHasher = Hasher<H> &&
+    requires (H& h) {
```

Comment:
BytesHasher evokes something that hashes bytes, not a hasher that returns bytes.